packages/react-components/src/components/VideoTile.tsx

```diff
@@ -295,6 +348,19 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
                   <Icon iconName="VideoTileMicOff" />
                 </Stack>
               )}
+              {
+                /* @conditional-compile-remove(pinned-participants) */
+                menuItems && menuItems.length > 0 && (
+                  <ControlBarButton
```

Comment:
ControlBarButton component is for use inside a ControlBar.
Let's not use it in the VideoTile.